packages/Organisms/ImageGallery/ImageGallery.vue

```diff
@@ -9,6 +9,7 @@
       :loop="true"
       :mouseDrag="false"
       :paginationEnabled="false"
+      @pageChange="onPageChange"
```

Comment:
This event `@pageChange="onPageChange"` will need to be added to the modal carousel as well (see line 38) in order for both fullscreen and small screen carousels to be in sync.

Reply:
Hi @alan-cole @tim-yao , 

I've updated the PR. 
Kindly review, 

Thank you.